translate/storage/mozilla_lang.py

```diff
@@ -91,7 +91,8 @@ def parse(self, lines):
                 readyTrans = False  # We already have our translation
                 continue
 
-            if line.startswith('#'):  # A comment
+            if line.startswith('#') and not line.startswith('##'):
```

Comment:
If comments always start with `# ` (note the space) then it might be easier to alter the condition to:

```python
            if line.startswith('# '):
                # Read comments, but not meta tags (e.g. '## TAG')
```